lib/osf-components/addon/components/registries/my-registrations-list/drafts/template.hbs

```diff
@@ -0,0 +1,25 @@
+<ContentPlaceholders as |placeholder|>
+    <PaginatedList::All
+        @modelName='draft-registration'
+        @usePlaceholders={{false}}
+        as |list|
+    >
+        <list.item as |draftRegistration|>
+            {{#if draftRegistration}}
+                <DraftRegistrationCard
+                    @draftRegistration={{draftRegistration}}
+                    @onDelete={{action list.doReload 1}}
+                />
+            {{else}}
+                {{placeholder.text lines=1}}
+            {{/if}}
+        </list.item>
+        <list.empty>
+            <p
+                data-test-draft-list-none
+            >
+                {{t 'registries.myRegistrationsList.noRegistrations'}}
```

Comment:
It looks like this should be fetching `{{t 'registries.myRegistrationsList.noDrafts'}}`